### ISSUE TITLE
Add instant close argument for NoticeMixin

### ISSUE THREAD

**Issue by marcebdev** [buefy/buefy#3064]
### Description

Add an argument to the close function such as

```js
close(instant = false) {
  ...
}
```

Which allows you to skip the close animation.

### Why Buefy need this feature

This is particularly important when you're scheduling different notifications and you need to programmatically override the previous one immediately.

I have not played around with the existsing `queue` prop available for *Toasts* and *Snackbars* as even if the queue prop could solve this issue the notification you're trying to close might be out of the scope of the current component. This means while `close()` can be done anywhere as long as a reference to the notification object is present (options such as event buses and global state exist for complex scheduling) queue cannot be managed in the same way.

**Comment by jtommy** [comment]
Maybe you can use $destroy() method. Can you try it ?

**Comment by marcebdev** [comment]
Yes, will re-provoke the behavior and test this in the next couple days and report back.

**Comment by marcebdev** [comment]
Did some testing, as far as I can tell the `$destroy()` method is not accessible from a toast's reference. I've set up a [CodePen](https://codepen.io/MarceDev/pen/oNzxWeK?editors=101) as a quite basic instance of this. In practice, the animation change is a bit more jarring.

On my project, I've used a timeout to prevent the loading toast from displaying immediately to prevent instances where an error is returned right away, this helps. In reality, it's difficult to determine how much of an improvement skipping the close animation would offer as there are lots of random variables in handling such a case. I'd imagine it would at least make the transition animation less jarring as it would kill the first toast sooner so it's likely worth a try.
